scripts/thyra-installer.py

```diff
@@ -0,0 +1,132 @@
+import urllib.request
+import subprocess
+import shutil
+import os
+import zipfile
+import ctypes
+from urllib.error import URLError
+from tarfile import ReadError
+
+# This file is to be bundled with pyinstaller in order to produce a .exe that can run on Windows without Python installed.
```

Comment:
```suggestion
# TODO: This file is to be bundled with pyinstaller in order to produce a .exe that can run on Windows without Python installed.
```

Reply:
This is not something that is left to do, it's already done, the .exe file contained in the PR is the bundled file.
I put this comment just to emphasize the fact that this python file is not meant to be used on its own.
Maybe I should explicitly state in the comment that in order to make a new release of our automatic installation process for Windows, this operation needs to be repeated to produce an updated .exe file.

Reply:
Understood. But I think we should use the CD script to create a binary file on the fly.